decidim-proposals/lib/tasks/proposals/upgrade/decdim_proposal_upgrade_tasks.rake

```diff
@@ -29,6 +29,20 @@ namespace :decidim do
             .destroy_all
         end
       end
+
+      desc "Migrates the withdrawn fields on proposals"
+      task migrate_proposal_withdrawn_fields: :environment do
```

Comment:
Since I have used the alternative path, the Release notes patch is not required. Fix implemented in [910f474](https://github.com/decidim/decidim/pull/12289/commits/910f474d8d70bac7973a88520e3ac8ae19ea7210)